server/controllers/caseNotesController.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response } from 'express'
+import { PagedListQueryParams } from '../interfaces/prisonApi/pagedList'
+import { mapHeaderData } from '../mappers/headerMappers'
+import { PrisonerSearchService } from '../services'
+import CaseNotesService from '../services/caseNotesService'
+import PrisonApiRestClient from '../data/prisonApiClient'
+import { PrisonApiClient } from '../data/interfaces/prisonApiClient'
+
+/**
+ * Parse request for case notes page and orchestrate response
+ */
+export default class CaseNotesController {
+  private prisonerSearchService: PrisonerSearchService
+
+  private caseNotesService: CaseNotesService
+
+  private prisonApiClient: PrisonApiClient
+
+  constructor(clientToken: string) {
+    this.prisonerSearchService = new PrisonerSearchService(clientToken)
+    this.caseNotesService = new CaseNotesService(clientToken)
+    this.prisonApiClient = new PrisonApiRestClient(clientToken)
+  }
+
+  public async displayCaseNotes(req: Request, res: Response) {
+    // Parse query params for paging, sorting and filtering data
+    const queryParams: PagedListQueryParams = {}
+    if (req.query.page) queryParams.page = +req.query.page
```

Comment:
Is that + there intentionally?

Reply:
Yeah, casts it to a number

Reply:
ah - javascript type coercion :v always fun